base/freestanding/femr/docker-compose.yaml

```diff
@@ -33,7 +33,7 @@ services:
 
 
   keycloak:
-    image: jboss/keycloak:9.0.0
+    image: jboss/keycloak:${KEYCLOAK_IMAGE_TAG:-15.0.1}
```

Comment:
Latest Keycloak version is 15.0.2. They shipped 15.0.1 and 15.0.2 on the same day.

Reply:
Thanks! fixed